e2e-tests/package.json

```diff
@@ -5,8 +5,8 @@
   "author": "The Matter Labs Team <hello@matterlabs.dev>",
   "license": "MIT",
   "devDependencies": {
-    "@matterlabs/hardhat-zksync-deploy": "^0.6.3",
-    "@matterlabs/hardhat-zksync-solc": "^0.4.0",
+    "@matterlabs/hardhat-zksync-deploy": "^0.6.5",
+    "@matterlabs/hardhat-zksync-solc": "^0.4.2",
```

Comment:
ah shoot, forgot to mention that we'll need to also update the `yarn.lock` file

Reply:
Doh should've known